ocs_ci/deployment/vmware.py

```diff
@@ -2046,10 +2080,17 @@ def clone_openshift_installer():
                     branch="release-4.12",
                 )
             else:
+                if config.DEPLOYMENT.get("ipv6"):
+                    constants.VSPHERE_INSTALLER_REPO = (
+                        "https://gitlab.cee.redhat.com/srozen/installer_ipv6.git"
```

Comment:
As discussed offline, I prefer to use public repo from github.
Also maintain the same branches per release and not hardcode master should be a case here as we know from the past that there were some changes between OCP versions.

As it's just cherry-pick I am not requesting changes now in this PR , but will be good to do changes in some follow up and cherry-pick to all versions where we did cherry-picked this change.